build.psm1

```diff
@@ -1571,6 +1571,8 @@ function Start-PSxUnit {
 
```

Comment:
@KirkMunro, your last commit had 32 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

TestDebugJobIPC
```powershell
FAIL A parameter cannot be found that matches parameter name 'BreakAll'.:TestInvokeError
Expected: True
Actual:   False
   at PSTests.Sequential.LocalDebuggerTests.TestDebugJobIPC() in /home/vsts/work/1/s/test/xUnit/csharp/test_Debugger.cs:line 207
```
TestDebugJobWinRM
```powershell
FAIL MI_RESULT_ACCESS_DENIED\u0000:TestInvokeError
Expected: True
Actual:   False
   at PSTests.Sequential.LocalDebuggerTests.TestDebugJobWinRM() in /home/vsts/work/1/s/test/xUnit/csharp/test_Debugger.cs:line 258
```
TestRemoteDebuggerBreak
```powershell
System.AggregateException : One or more errors occurred. (MI_RESULT_ACCESS_DENIED\u0000) (The term 'Restart-Service' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.)
---- System.Management.Automation.PSInvalidOperationException : MI_RESULT_ACCESS_DENIED\u0000
---- System.Management.Automation.CommandNotFoundException : The term 'Restart-Service' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

----- Inner Stack Trace #1 (System.Management.Automation.PSInvalidOperationException) -----
   at System.Management.Automation.Remoting.Client.WSManClientSessionTransportManager.Initialize(Uri connectionUri, WSManConnectionInfo connectionInfo) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs:line 1506
   at System.Management.Automation.Remoting.Client.WSManClientSessionTransportManager..ctor(Guid runspacePoolInstanceId, WSManConnectionInfo connectionInfo, PSRemotingCryptoHelper cryptoHelper, String sessionName) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs:line 586
   at System.Management.Automation.Runspaces.WSManConnectionInfo.CreateClientSessionTransportManager(Guid instanceId, String sessionName, PSRemotingCryptoHelper cryptoHelper) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs:line 1087
   at System.Management.Automation.Remoting.ClientRemoteSessionDSHandlerImpl..ctor(ClientRemoteSession session, PSRemotingCryptoHelper cryptoHelper, RunspaceConnectionInfo connectionInfo, URIDirectionReported uriRedirectionHandler) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/remotingprotocolimplementation.cs:line 82
   at System.Management.Automation.Remoting.ClientRemoteSessionImpl..ctor(RemoteRunspacePoolInternal rsPool, URIDirectionReported uriRedirectionHandler) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/clientremotesession.cs:line 214
   at System.Management.Automation.Internal.ClientRunspacePoolDataStructureHandler.CreateClientRemoteSession(RemoteRunspacePoolInternal rsPoolInternal) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/RemotingProtocol2.cs:line 473
   at System.Management.Automation.Internal.ClientRunspacePoolDataStructureHandler..ctor(RemoteRunspacePoolInternal clientRunspacePool, TypeTable typeTable) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/RemotingProtocol2.cs:line 42
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal.CreateDSHandler(TypeTable typeTable) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/RemoteRunspacePoolInternal.cs:line 172
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal..ctor(Int32 minRunspaces, Int32 maxRunspaces, TypeTable typeTable, PSHost host, PSPrimitiveDictionary applicationArguments, RunspaceConnectionInfo connectionInfo, String name) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/RemoteRunspacePoolInternal.cs:line 97
   at System.Management.Automation.Runspaces.RunspacePool..ctor(Int32 minRunspaces, Int32 maxRunspaces, TypeTable typeTable, PSHost host, PSPrimitiveDictionary applicationArguments, RunspaceConnectionInfo connectionInfo, String name) in /home/vsts/work/1/s/src/System.Management.Automation/engine/hostifaces/RunspacePool.cs:line 606
   at System.Management.Automation.RemoteRunspace..ctor(TypeTable typeTable, RunspaceConnectionInfo connectionInfo, PSHost host, PSPrimitiveDictionary applicationArguments, String name, Int32 id) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/remoterunspace.cs:line 143
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(RunspaceConnectionInfo connectionInfo, PSHost host, TypeTable typeTable, PSPrimitiveDictionary applicationArguments, String name) in /home/vsts/work/1/s/src/System.Management.Automation/engine/hostifaces/ConnectionFactory.cs:line 557
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(RunspaceConnectionInfo connectionInfo, PSHost host, TypeTable typeTable) in /home/vsts/work/1/s/src/System.Management.Automation/engine/hostifaces/ConnectionFactory.cs:line 495
   at PSTests.Sequential.RemoteDebuggerTests.InitializeRunspace() in /home/
```
TestRemoteDebuggerProcessCommand
```powershell
System.AggregateException : One or more errors occurred. (MI_RESULT_ACCESS_DENIED\u0000) (The term 'Restart-Service' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.)
---- System.Management.Automation.PSInvalidOperationException : MI_RESULT_ACCESS_DENIED\u0000
---- System.Management.Automation.CommandNotFoundException : The term 'Restart-Service' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

----- Inner Stack Trace #1 (System.Management.Automation.PSInvalidOperationException) -----
   at System.Management.Automation.Remoting.Client.WSManClientSessionTransportManager.Initialize(Uri connectionUri, WSManConnectionInfo connectionInfo) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs:line 1506
   at System.Management.Automation.Remoting.Client.WSManClientSessionTransportManager..ctor(Guid runspacePoolInstanceId, WSManConnectionInfo connectionInfo, PSRemotingCryptoHelper cryptoHelper, String sessionName) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs:line 586
   at System.Management.Automation.Runspaces.WSManConnectionInfo.CreateClientSessionTransportManager(Guid instanceId, String sessionName, PSRemotingCryptoHelper cryptoHelper) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs:line 1087
   at System.Management.Automation.Remoting.ClientRemoteSessionDSHandlerImpl..ctor(ClientRemoteSession session, PSRemotingCryptoHelper cryptoHelper, RunspaceConnectionInfo connectionInfo, URIDirectionReported uriRedirectionHandler) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/remotingprotocolimplementation.cs:line 82
   at System.Management.Automation.Remoting.ClientRemoteSessionImpl..ctor(RemoteRunspacePoolInternal rsPool, URIDirectionReported uriRedirectionHandler) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/clientremotesession.cs:line 214
   at System.Management.Automation.Internal.ClientRunspacePoolDataStructureHandler.CreateClientRemoteSession(RemoteRunspacePoolInternal rsPoolInternal) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/RemotingProtocol2.cs:line 473
   at System.Management.Automation.Internal.ClientRunspacePoolDataStructureHandler..ctor(RemoteRunspacePoolInternal clientRunspacePool, TypeTable typeTable) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/RemotingProtocol2.cs:line 42
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal.CreateDSHandler(TypeTable typeTable) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/RemoteRunspacePoolInternal.cs:line 172
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal..ctor(Int32 minRunspaces, Int32 maxRunspaces, TypeTable typeTable, PSHost host, PSPrimitiveDictionary applicationArguments, RunspaceConnectionInfo connectionInfo, String name) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/RemoteRunspacePoolInternal.cs:line 97
   at System.Management.Automation.Runspaces.RunspacePool..ctor(Int32 minRunspaces, Int32 maxRunspaces, TypeTable typeTable, PSHost host, PSPrimitiveDictionary applicationArguments, RunspaceConnectionInfo connectionInfo, String name) in /home/vsts/work/1/s/src/System.Management.Automation/engine/hostifaces/RunspacePool.cs:line 606
   at System.Management.Automation.RemoteRunspace..ctor(TypeTable typeTable, RunspaceConnectionInfo connectionInfo, PSHost host, PSPrimitiveDictionary applicationArguments, String name, Int32 id) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/remoterunspace.cs:line 143
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(RunspaceConnectionInfo connectionInfo, PSHost host, TypeTable typeTable, PSPrimitiveDictionary applicationArguments, String name) in /home/vsts/work/1/s/src/System.Management.Automation/engine/hostifaces/ConnectionFactory.cs:line 557
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(RunspaceConnectionInfo connectionInfo, PSHost host, TypeTable typeTable) in /home/vsts/work/1/s/src/System.Management.Automation/engine/hostifaces/ConnectionFactory.cs:line 495
   at PSTests.Sequential.RemoteDebuggerTests.InitializeRunspace() in /home/
```
TestRemoteDebuggerCommandBreakpointUpdated
```powershell
System.AggregateException : One or more errors occurred. (MI_RESULT_ACCESS_DENIED\u0000) (The term 'Restart-Service' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.)
---- System.Management.Automation.PSInvalidOperationException : MI_RESULT_ACCESS_DENIED\u0000
---- System.Management.Automation.CommandNotFoundException : The term 'Restart-Service' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

----- Inner Stack Trace #1 (System.Management.Automation.PSInvalidOperationException) -----
   at System.Management.Automation.Remoting.Client.WSManClientSessionTransportManager.Initialize(Uri connectionUri, WSManConnectionInfo connectionInfo) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs:line 1506
   at System.Management.Automation.Remoting.Client.WSManClientSessionTransportManager..ctor(Guid runspacePoolInstanceId, WSManConnectionInfo connectionInfo, PSRemotingCryptoHelper cryptoHelper, String sessionName) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs:line 586
   at System.Management.Automation.Runspaces.WSManConnectionInfo.CreateClientSessionTransportManager(Guid instanceId, String sessionName, PSRemotingCryptoHelper cryptoHelper) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs:line 1087
   at System.Management.Automation.Remoting.ClientRemoteSessionDSHandlerImpl..ctor(ClientRemoteSession session, PSRemotingCryptoHelper cryptoHelper, RunspaceConnectionInfo connectionInfo, URIDirectionReported uriRedirectionHandler) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/remotingprotocolimplementation.cs:line 82
   at System.Management.Automation.Remoting.ClientRemoteSessionImpl..ctor(RemoteRunspacePoolInternal rsPool, URIDirectionReported uriRedirectionHandler) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/clientremotesession.cs:line 214
   at System.Management.Automation.Internal.ClientRunspacePoolDataStructureHandler.CreateClientRemoteSession(RemoteRunspacePoolInternal rsPoolInternal) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/RemotingProtocol2.cs:line 473
   at System.Management.Automation.Internal.ClientRunspacePoolDataStructureHandler..ctor(RemoteRunspacePoolInternal clientRunspacePool, TypeTable typeTable) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/RemotingProtocol2.cs:line 42
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal.CreateDSHandler(TypeTable typeTable) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/RemoteRunspacePoolInternal.cs:line 172
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal..ctor(Int32 minRunspaces, Int32 maxRunspaces, TypeTable typeTable, PSHost host, PSPrimitiveDictionary applicationArguments, RunspaceConnectionInfo connectionInfo, String name) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/RemoteRunspacePoolInternal.cs:line 97
   at System.Management.Automation.Runspaces.RunspacePool..ctor(Int32 minRunspaces, Int32 maxRunspaces, TypeTable typeTable, PSHost host, PSPrimitiveDictionary applicationArguments, RunspaceConnectionInfo connectionInfo, String name) in /home/vsts/work/1/s/src/System.Management.Automation/engine/hostifaces/RunspacePool.cs:line 606
   at System.Management.Automation.RemoteRunspace..ctor(TypeTable typeTable, RunspaceConnectionInfo connectionInfo, PSHost host, PSPrimitiveDictionary applicationArguments, String name, Int32 id) in /home/vsts/work/1/s/src/System.Management.Automation/engine/remoting/client/remoterunspace.cs:line 143
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(RunspaceConnectionInfo connectionInfo, PSHost host, TypeTable typeTable, PSPrimitiveDictionary applicationArguments, String name) in /home/vsts/work/1/s/src/System.Management.Automation/engine/hostifaces/ConnectionFactory.cs:line 557
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(RunspaceConnectionInfo connectionInfo, PSHost host, TypeTable typeTable) in /home/vsts/work/1/s/src/System.Management.Automation/engine/hostifaces/ConnectionFactory.cs:line 495
   at PSTests.Sequential.RemoteDebuggerTests.InitializeRunspace() in /home/
```